### ISSUE TITLE
Do I need qemu?

### ISSUE THREAD

**Issue by Mirage3098** [yusufhanoglu/IsolatedOS#1]
Hello! I love the look of your Alien OS, but when I try to mount the ISO file, it says it won't work. I'm not the best at coding, but could you help me on how to do it? Do I need qemu or something else?

**Comment by hanoglu** [comment]
Yes, you can use any virtual machine environment. Qemu might be one of them.

**Comment by Mirage3098** [comment]
Thanks! Do you reckon you could make it an app or something? (I'm not too
expert with virtual machines) Thanks again! :)
